prio/server.c

```diff
@@ -83,6 +83,46 @@ PrioServer_aggregate(PrioServer s, PrioVerifier v)
   return MPArray_addmod(s->data_shares, arr, &s->cfg->modulus);
 }
 
+int
```

Comment:
Should these two functions be declared `static int`?

Reply:
Ah yes, it's been a while since I've written a significant amount of C. This effectively makes the function private in the scope of the file, which is the intent.